packages/envied_generator/lib/src/generator.dart

```diff
@@ -84,8 +86,10 @@ final class EnviedGenerator extends GeneratorForAnnotation<Envied> {
         ]),
     );
 
-    const String ignore = '// coverage:ignore-file\n'
-        '// ignore_for_file: type=lint';
+    final String ignore = '''
```

Comment:
Could you please keep string literal style here and use something along the lines of
```dart
const String ignore = '// coverage:ignore-file\n'
         '// ignore_for_file: type=lint\n'
         '// generated_from: ${config.path}';
```

Reply:
Sure.